frame/merkle-mountain-range/primitives/src/lib.rs

```diff
@@ -429,6 +429,9 @@ sp_api::decl_runtime_apis! {
 		/// The leaf data is expected to be encoded in it's compact form.
 		fn verify_proof_stateless(root: Hash, leaf: EncodableOpaqueLeaf, proof: Proof<Hash>)
 			-> Result<(), Error>;
+
+		/// Return the on-chain MMR root hash.
+		fn mmr_root() -> Result<Hash, Error>;
```

Comment:
Does adding this new fn to the runtime api require to also version the api declaration (`#[api_version(x)]`)?

Reply:
I think that shouldn't be necessary.